### DISCUSSION TITLE
Is ZuluSCSI firmware compatible with SCSI2SD?

### DISCUSSION THREAD

**Discussion by MM502** [ZuluSCSI/ZuluSCSI-firmware#9] [Q&A] [answered]
Hello,
is it possible to flash the ZuluSCSI firmware to existing SCSI2SD drives?
Thank you.

**Comment by PetteriAimonen** [comment, marked as answer]
Currently there is no support for SCSI2SD hardware. But the aim of the firmware is to be very portable so SCSI2SD hardware support could potentially be added at some point.
